packages/cactus-core/src/main/typescript/public-api.ts

```diff
@@ -4,3 +4,4 @@ export {
   ConsortiumRepository,
   IConsortiumRepositoryOptions,
 } from "./consortium-repository";
+export * from "./consensus-has-transaction-finality";
```

Comment:
@jscode017 Recommendation here is to explicitly export not with the `*` character (easier to read what exactly goes in the public API that way because then there's no need to open the file up itself and skim through it's exports).
There's exceptions to this practice when there's a lot of exports for example in a generated file, but here it's just the one so I'd say it's easier on the reader if you export it explicitly

```typescript
export { theFunction } from "./consensus-has-transaction-finality";
```